firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigLogger.java

```diff
@@ -0,0 +1,117 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.remoteconfig.internal;
+
+import android.util.Log;
+
+/**
+ * A per-namespace logger with settable log level. Defaults to level {@link Log#INFO}.
+ *
+ * @author danasilver
+ */
+public class ConfigLogger {
+  private static final String TAG = "FirebaseRemoteConfig";
+
+  private final String tag;
+  private int logLevel;
+
+  private ConfigLogger(String tag) {
+    this.tag = tag;
+    this.logLevel = Log.INFO;
+  }
+
+  /** Get the logger for {@param namespace} or create one if it does not already exist. */
+  public static ConfigLogger getLogger() {
+    return new ConfigLogger(TAG);
```

Comment:
Given `TAG` is effectively constant, I don't see a reason to parameterize it, and if we don't have any params, I think it's ok to just call `new ConfigLogger()` in a few places.

If we need to know what the tag is in tests, I think it's ok to make it public (esp since we're in the internal package) and visible for testing, or leave it parameterized and just have a no-arg constructor overload.

If a creation method is handy, I think we should name it something that implies creation, eg `ConfigLogger.create()`. `getLogger` seems like `getInstance`, but it isn't a singleton.

Reply:
Good call, that simplifies things.